src/main/java/Duke.java

```diff
@@ -7,52 +7,87 @@ public static void main(String[] args) {
                 + "|  o  /|     ||  o  /|     |\n"
                 + "|  o  \\|  o  ||  o  \\|  o  |\n"
                 + "|_____/|_____||_____/|_____|\n";
-        String HORIZONTAL_LINE = "____________________________________________________________";
+        String horizontalLine = "____________________________________________________________";
```

Comment:
For constants, I believe you can use the previous naming format which you used (HORIZONTAL_LINE)